src/app/state/rest/cases.ts

```diff
@@ -5,37 +5,52 @@ import { useErrorHandler } from "./hooks/utils/errorHandler"
 import { makeApiUrl } from "./hooks/utils/apiUrl"
 import useApiRequest from "./hooks/useApiRequest"
 
+
+const sortingOrder = {
+  ASCEND: "ASCEND",
+  DESCEND: "DESCEND"
+}
+
+const dataIndexMapping: any = {
+  "address.full_address": "address__street_name",
+  "last_updated": "last_updated"
+}
+
+const getOrderingValue = (sorting: TABLE.Schemas.Sorting) => {
+  let value = ""
+  if (sorting?.dataIndex) {
+    value = dataIndexMapping[sorting.dataIndex]
+  }
+  if (sorting.order === sortingOrder.DESCEND) {
+    value = `-${ value }`
+  }
+  return value
+}
+
 export const useCases = (
-  page: number,
-  page_size: number,
-  theme: string,
   sensitive = false,
-  from_start_date?: string,
-  ordering?: string,
+  pagination: TABLE.Schemas.Pagination,
+  sorting?: TABLE.Schemas.Sorting,
+  theme?: string,
+  fromStartDate?: string,
   options?: Options
 ) => {
   const handleError = useErrorHandler()
-  const urlParams: any = {}
-  if (page !== undefined) {
-    urlParams.page = page
-  }
-  if (page_size !== undefined) {
-    urlParams.page_size = page_size
+  const urlParams: any = {
+    page: pagination.page,
+    page_size: pagination.pageSize
   }
   if (sensitive === false) {
     urlParams.sensitive = false
   }
-  if (theme) {
+  if (theme !== undefined) {
     urlParams.theme = theme
   }
-  if (from_start_date !== undefined) {
-    urlParams.from_start_date = from_start_date
+  if (fromStartDate !== undefined) {
```

Comment:
Check if this is necessary, by default `stringify` ignores undefined attributes.